ext/cairo/rb_cairo_font_face.c

```diff
@@ -121,6 +121,46 @@ cr_font_face_quartz_supported_p (VALUE klass)
 #endif
 }
 
+#if CAIRO_HAS_FT_FONT
+static void
+handle_ft_error(FT_Error error)
+{
+#undef __FTERRORS_H__
```

Comment:
Sorry. I didn't know the recommendation. `#undef __FTERRORS_H__` was too strange for me.